code/datums/datumvars.dm

```diff
@@ -386,7 +386,10 @@ body
 				html += "<ul>"
 				var/index = 1
 				for(var/entry in L)
-					html += debug_variable(index, L[index], level + 1)
+					if(istext(entry))

```

Comment:
Wait, what if it's legit `null` there, like `organs_by_name` when the limb is missing?